### ISSUE TITLE
sipml5 test automation

### ISSUE THREAD

**Issue by moinahmed** [DoubangoTelecom/sipml5#233]
Hi,

I want to automate load testing of my webrtc2sip server which I connect through sipml5.
Currently we do it manually by opening so many browser instances and then connect through sipml5 clients.
Can anyone guide me on this.

Thanks.

**Comment by vferko** [comment]
You could use Selenium for it. The easiest way to use it is through Robot Framework, if you are not experienced in other automated testing solutions.

**Comment by moinahmed** [comment]
Can we do it through command line? The main aim is to test our server instead of client.

**Comment by vferko** [comment]
Yes, the tests are run from command line. Easy to integrate with CircleCI or other CI systems.

**Comment by moinahmed** [comment]
Yeah, but I do not want to do it through my web client automation, which is the case if I do through Selenium . I am looking for some python script or any other way to code it in my program and send it. So if you could tell me any webrtc lib (written in python/java/c++) through which I can simulate audio calls.

**Comment by ajmorenodelarosa** [comment]
You can write some useful tests using freeswitch and python. Also you can use ESL https://wiki.freeswitch.org/wiki/Event_Socket_Library with Java. I use it already to generate calls with php.

**Comment by moinahmed** [comment]
any webrtc library for python/java?
